### DISCUSSION TITLE
How do I use Martini 3 with GROMACS?

### DISCUSSION THREAD

**Discussion by CeraveWasTakenLOL** [Martini-Force-Field-Initiative/MartiniDiscussions#7] [General]
I downloaded Martini 3 and moved it to my force field directory but GROMACS doesn't recognize it. This has not been an issue with any other force field I've downloaded.

**Comment by csbrasnett** [comment]
While Martini has been (principally) optimised to be run with gromacs, this doesn't mean that Martini is native to gromacs. For example pdb2gmx won't be able to just make a protein structure, because the atom names in rtp files don't correspond to martini bead names. For this purpose, the martini community develops tools like [martinize2](https://github.com/marrink-lab/vermouth-martinize/) to generate coarse-grained topology files as inputs that gromacs can read (in short, generate tpr files to run simulations with). The collection of [tutorials](https://cgmartini.nl/docs/tutorials/Martini3/tutorials.html) that we've developed might be a good place to start to see how this can work in practice if you're already familiar with gromacs.
